msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -341,7 +342,10 @@ Use the following example to configure your app manifest's `webApplicationInfo`
 </details>
 
 > [!NOTE]
-> The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
+>
+> * The bot can receive meeting start or end events automatically from all the meetings created in all the channels by adding `ChannelMeeting.ReadBasic.Group` to manifest for RSC permission.
```

Comment:
Below we say
> Currently apps aren't supported in scheduled and instant meetings in Teams channels.

The messaging is mixed here-- the statement below may need some more explanation.